db/migrate/20210128021147_add_allow_unknown_sender_topic_replies_to_group.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddAllowUnknownSenderTopicRepliesToGroup < ActiveRecord::Migration[6.0]
+  def up
+    add_column :groups, :allow_unknown_sender_topic_replies, :boolean, default: false
+    DB.exec("UPDATE groups SET allow_unknown_sender_topic_replies = false")
```

Comment:
Since there is a default of `false` this update should be pointless?

I also recommend making the column `null: false`